src/tictactoe/board_translators.clj

```diff
@@ -4,13 +4,18 @@
 
 (defn inverse-ui-mapping [idx] (- (Integer/parseInt idx) 1))
 
-(defn player-mapping [itm]
-  (get {1 "X" -1 "O"} itm))
+;(defn ui-mapping [idx] (str (char (+ idx 97))))
```

Comment:
Renamed & uncommented in 6aa733d. Wanted to allow for changing the board mapping to letters for example.